package.json

```diff
@@ -10,14 +10,15 @@
     "url": "https://github.com/ipython/ipywidgets.git"
   },
   "scripts": {
+    "postinstall": "npm run bower",
```

Comment:
Wait, now that I think some more, I *don't* think this is needed.  We package the compiled CSS which is all the bower dependencies are used for, no?